.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -2,6 +2,7 @@ name: Test
 
 on:
   pull_request:
+    types: [opened, edited, labeled, unlabeled, converted_to_draft, ready_for_review]
```

Comment:
Why do we need to be so specific? Can't this be triggered just with `pull_request`?

Reply:
Thanks for catching the issue. Come to think of it, we don't really need an immediate workflow trigger after a PR state change, as no code has been changed.

And, just `pull_request` gets the job done where we make an existing open PR WIP and push commits in it; for each subsequent commit, it skips danger.

Fixed in d5fc88d7a3d991bf619fda410262669c31c03a38